smdebug/tensorflow/collection.py

```diff
@@ -114,6 +114,11 @@ def set_tensor_ref(self, tensor):
         self._tensors[tensor.name] = tensor
         self.add_tensor_name(tensor.export_name)
 
+    def set_tensor(self, tensor_name, tensor):
```

Comment:
More descriptive method name and docstring - unclear what is being set.

Reply:
its set_tensor_ref but without tensor_Ref usage.
just assign the tensor as is